dot/core/interface.go

```diff
@@ -70,7 +70,8 @@ type BlockState interface {
 	SubChain(start, end common.Hash) ([]common.Hash, error)
 	GetBlockBody(hash common.Hash) (*types.Body, error)
 	HandleRuntimeChanges(newState *rtstorage.TrieState, in runtime.Instance, bHash common.Hash) error
-	GetRuntime(*common.Hash) (runtime.Instance, error)
+	GetBestBlockRuntime() runtime.Instance
+	GetRuntime(hash *common.Hash) (instance runtime.Instance, err error)
```

Comment:
Why doesn't `GetBlockBlockRuntime()` return an error?

Reply:
I had this returning an error, but updated based on this comment: https://github.com/ChainSafe/gossamer/pull/2781#discussion_r973095859